tools/install_opte.sh

```diff
@@ -122,15 +122,15 @@ function add_publisher {
 # `helios-netdev` provides the xde kernel driver and the `opteadm` userland tool
 # for interacting with it.
 HELIOS_NETDEV_BASE_URL="https://buildomat.eng.oxide.computer/public/file/oxidecomputer/opte/repo"
-HELIOS_NETDEV_COMMIT="23884d35aa7908e23accaa77f125a370ddf5c606"
+HELIOS_NETDEV_COMMIT="52176b99cc578f5b7b90f9bae8935013f961086c"
```

Comment:
We should also pull this into `sled-agent/Cargo.toml`, on lines 58 and 59. I don't think we'd get a version mismatch in the `ioctl(2)` calls, but I've been trying to keep these from drifting. (We could consider grabbing this out of that Cargo.toml to avoid duplication.)